resources/Materials/Examples/StandardSurface/standard_surface_default.mtlx

```diff
@@ -2,6 +2,45 @@
 <materialx version="1.36">
   <material name="Default">
     <shaderref name="SR_default" node="standard_surface">
+      <bindinput name="base" type="float" value="0.8"/>
```

Comment:
Just curious why we need to have explicit bind inputs here as this is the "default". Or is it you want to be able to edit all the standard surface inputs?

Reply:
Yes exactly, this was the only way I could find to make the inputs show up in the viewer property editor :)

Reply:
Sounds good. It's probably good to have access to all the inputs on standard surface for playing around with.